src/runner/mod.rs

```diff
@@ -96,6 +78,20 @@ impl Runner {
             }
         }
     }
+
+    fn prepare_before_run(&self) {
+        panic::set_hook(Box::new(|_panic_info| {
+            // XXX panics already catched at the test call site, don't output the trace in stdout
+        }));
+        let threads = if self.configuration.parallel { 0 } else { 1 }; // 0 is rayon default
+        let rayon_config = rayon::Configuration::new().num_threads(threads);
+        let _ = rayon::initialize(rayon_config);
```

Comment:
Reasoning being that more than a single runner can be created via rspec, each calling into `rayon::initialize(…)`. 🤔